exporter/clickhousetracesexporter/migrations/000014_add_usage_explorer_mv.down.sql

```diff
@@ -0,0 +1,2 @@
+DROP TABLE IF EXISTS signoz_traces.usage_explorer
+DROP VIEW IF EXISTS signoz_traces.usage_explorer_mv
```

Comment:
```suggestion
DROP TABLE IF EXISTS signoz_traces.usage_explorer;
DROP VIEW IF EXISTS signoz_traces.usage_explorer_mv;
```

Never tried migrate down. Are semicolons required?

Reply:
Thanks for pointing out, down is failing even before usage explorer.
Creating a separate PR for the same.

Reply:
Please review this PR: https://github.com/SigNoz/opentelemetry-collector-contrib/pull/2074